core/creators/themedui.js

```diff
@@ -287,7 +287,9 @@ CKEDITOR.replaceClass = 'ckeditor';
 			outer = container;
 		}
 
-		width = convertCssUnitToPx( width );
+		if ( width ) {
```

Comment:
It will also skip conversion for 0. Since 0 can be used without units (`px` in this case) it will work fine (kind of as a side effect). However, lack of such explicitness may lead to difficulties in understanding of this code (and it's intentions) in the future. Therefore I would be for checking `width || width === 0`. WDYT?